src/assets/css/App.css

```diff
@@ -232,9 +235,28 @@ header .menuIcon {
   background-size: 32px;
   background-repeat: no-repeat;
   cursor: pointer;
+}
+
+.food {
   background-image: url("../img/shopping.png");
 }
 
+.medicine {
```

Comment:
Those classes are there for displaying different icons depending on which category is being rendered. This should help setting the styles for the different selectable pins.